<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Seznamy závodníků</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>449</width>
            <height>167</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Startovní listina</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_readFile">
             <property name="text">
              <string>Načti CSV</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_printStartList">
             <property name="text">
              <string>Tisk</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="tv_startList">
             <property name="acceptDrops">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="sa_results">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>449</width>
            <height>167</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Výsledná listina</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_loadResults">
             <property name="text">
              <string>Načti CSV</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_printResults">
             <property name="text">
              <string>Tisk</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="tv_results"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Nastavení</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Start závodu 10 km: </string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QTimeEdit" name="te_startTime">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="dateTime">
               <datetime>
                <hour>0</hour>
                <minute>0</minute>
                <second>0</second>
                <year>2000</year>
                <month>1</month>
                <day>1</day>
               </datetime>
              </property>
              <property name="displayFormat">
               <string>HH:mm:ss</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="lbl_current10">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>00:00:00</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_start">
            <property name="text">
             <string>START 10 km</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Start závodu 4 km:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QTimeEdit" name="te_startTime4">
              <property name="displayFormat">
               <string>HH:mm:ss</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="lbl_current4">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>00:00:00</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_start4">
            <property name="text">
             <string>START 4 km</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Start dětského závodu:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QTimeEdit" name="te_startTimeChildren">
              <property name="displayFormat">
               <string>HH:mm:ss</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="lbl_currentChildren">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>00:00:00</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_startChildren">
            <property name="text">
             <string>START dětského závodu</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="lbl_time">
              <property name="font">
               <font>
                <pointsize>24</pointsize>
               </font>
              </property>
              <property name="text">
               <string>00:00:00</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Přidat výsledek</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLineEdit" name="le_runnerID">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>28</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>IBeamCursor</cursorShape>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>Číslo závodníka</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export...</string>
    </property>
    <widget class="QMenu" name="menuPodle_trati">
     <property name="title">
      <string>Podle trati</string>
     </property>
     <addaction name="action10_km"/>
     <addaction name="action4_km"/>
    </widget>
    <addaction name="menuPodle_trati"/>
    <addaction name="actionPodle_kategorie"/>
   </widget>
   <addaction name="menuExport"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPodle_kategorie">
   <property name="text">
    <string>Podle kategorie</string>
   </property>
  </action>
  <action name="action10_km">
   <property name="text">
    <string>10 km</string>
   </property>
  </action>
  <action name="action4_km">
   <property name="text">
    <string>4 km</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
